### PR TITLE
Remove react-test-renderer dependency

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

File: packages/yourloops/app/main-lobby.tsx [hunk 1]
```diff
@@ -65,46 +66,51 @@ const routeStyle = makeStyles<Theme>(() => {
   }
 })
 
+export const isRoutePublic = (route: string): boolean => {
+  return PUBLIC_ROUTES.includes(route)
+}
```
Comment (fatb38) [inline packages/yourloops/app/main-lobby.tsx:71]: Why export this function ? And what about a one liner ?
```suggestion
export const isRoutePublic = (route: string): boolean => PUBLIC_ROUTES.includes(route)
```

File: packages/yourloops/app/main-lobby.tsx [hunk 2]
```diff
@@ -65,46 +66,51 @@ const routeStyle = makeStyles<Theme>(() => {
...
-    }
-    if (!trainingPath && currentRoute !== COMPLETE_SIGNUP_PATH && !renewConsentPath && user && user.hasToDisplayTrainingInfoPage()) {
-      return '/training'
-    }
-    return null
-  }
-
-  const redirectTo = checkRedirect()
+  const style = isCurrentRoutePublic || currentRoute === COMPLETE_SIGNUP_PATH ? classes.public : classes.private
+  const redirectTo = getRedirectUrl(currentRoute, user, isAuthenticated)
```
Comment (fatb38) [inline packages/yourloops/app/main-lobby.tsx:113]: Nice 😉 really more readable component now

File: packages/yourloops/test/unit/app/main-lobby.test.tsx [hunk 3]
```diff
@@ -25,148 +25,94 @@
...
 import User from '../../../lib/auth/user'
-import PatientConsentPage from '../../../pages/patient/patient-consent'
-import DirectShareApi from '../../../lib/share/direct-share-api'
-import TrainingPage from '../../../pages/training/training'
-import LoginPage from '../../../pages/login/login-page'
+import { getRedirectUrl } from '../../../app/main-lobby'
 
-jest.mock('../../../lib/auth')
-jest.mock('@auth0/auth0-react')
 describe('Main lobby', () => {
```
Comment (fatb38) [inline packages/yourloops/test/unit/app/main-lobby.test.tsx:31]: 👏🏻 speed up !

**Review by fatb38 — CHANGES_REQUESTED** [review]

File: packages/yourloops/app/main-lobby.tsx (hunk 1 above)
Comment (TimCanu) [reply packages/yourloops/app/main-lobby.tsx:71]: Function is exported for tests. I don't like one liner

File: packages/yourloops/app/main-lobby.tsx (hunk 2 above)
Comment (TimCanu) [reply packages/yourloops/app/main-lobby.tsx:111]: yes :slightly_smiling_face:

File: packages/yourloops/test/unit/app/main-lobby.test.tsx (hunk 3 above)
Comment (TimCanu) [reply packages/yourloops/test/unit/app/main-lobby.test.tsx:31]: This was our longest unit test. Now it runs in 3ms

**Review by fatb38 — APPROVED** [review]

**Pull request merged**